app/routes/index.js

```diff
@@ -1,6 +1,34 @@
 import AuthenticatedRouteMixin from 'ember-simple-auth/mixins/authenticated-route-mixin';
+import Navigation from 'hospitalrun/mixins/navigation';
+import SetupUserRole from 'hospitalrun/mixins/setup-user-role';
+import UserRoles from 'hospitalrun/mixins/user-roles';
 import Ember from 'ember';
-export default Ember.Route.extend(AuthenticatedRouteMixin, {
+const { inject } = Ember;
+
+export default Ember.Route.extend(AuthenticatedRouteMixin, Navigation, SetupUserRole, UserRoles, {
+  session: inject.service(),
+  beforeModel() {
+    // this.setupUserRole();
+    let session = this.get('session');
+    if (session != null) {
+      let role = session.get('data.authenticated.userRole');
```

Comment:
This should pull data.authenticated.role not data.authenticated.userRole, particularly if you want the acceptance test I added to pass.  😄